catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -73,6 +73,8 @@
     private static final String GEOMETRY_SORT_FIELD =
             Metacard.GEOGRAPHY + SchemaFields.GEO_SUFFIX + SchemaFields.SORT_KEY_SUFFIX;
 
+    private static final String EXT_SORT_BY = "EXT_SORT_BY";
```

Comment:
❗️ Please make the constant value more readable.  For example, `additional.sort.bys` makes it clearer what the property will do if present.